src/api/buddies.ts

```diff
@@ -54,15 +54,31 @@ const banRequest = (
   accessToken: authApi.AccessToken,
   status: BanActions,
 ) => {
-  const statusStr = status === 'Ban' ? 'banned' : 'ok';
+  var statusStr = 'ok';
```

Comment:
`let` would be better.   and could it be typechecked too? so typescript would complain if given a typo eg 'baned' or 'delted'

Reply:
Sure. Btw, I don't know whether the if-else approach is a good solution for this, but I couldn't come up with anything better. What do you think?